docs/content/direct/release.md

```diff
@@ -11,7 +11,11 @@ clues about the problem.
 
 Making a new release requires a contributor to do the following things. Here `$version` is the semver identifier for the release (e.g., `1.2.3-rc2`).
 
-- Edit the tag in the default transport controller image setting (`export TRANSPORT_CONTROLLER_IMAGE...`) to the latest release of [ks/OTP](https://github.com/kubestellar/ocm-transport-plugin). **NOTE** This step is not really part of making the next ks/ks release, it is part of making that ks/OTP release. As such, the timing is driven by that release, not the ks/ks release.
+- Edit `scripts/deploy-transport-controller.sh`: update the tag in the default transport controller image setting (`export TRANSPORT_CONTROLLER_IMAGE...`) to the latest release of [ks/OTP](https://github.com/kubestellar/ocm-transport-plugin). **NOTE** This step is not really part of making the next ks/ks release, it is part of making that ks/OTP release. As such, the timing is driven by that release, not the ks/ks release.
```

Comment:
Maybe move these 3 points to a section of "Creating a new OTP release"  ?

Reply:
The force-push to 8a264da48 introduces such a section.